articles/aws-api-setup.md

```diff
@@ -1,70 +1,104 @@
-# How to Setup AWS to do Delegated Authentication with APIs
+# Setup AWS for Delegated Authentication with APIs
 
-In order to do delegated authentication with AWS APIs you have to create a **SAML Provider** and one or more **Roles**. This document explains how to do that.
+To enable delegated authentication with AWS APIs, you must create a **SAML Provider** and one or more **Roles**.
 
-## Creating the SAML Provider
+## Create a SAML Provider
 
-This is a one time setup
+To create a SAML provider, follow these steps:
 
-1. Go to the **AWS IAM Console** and click on **Identity Providers**
+1. Go to the [AWS IAM Console](https://console.aws.amazon.com/iam/home#home). Click on **Identity Providers** in the left menu and then click **Create Provider**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-1.png)
 
-2. Click on **Create SAML Provider** and give it a name.
+2. Select **SAML** as the **Provider Type** and enter a name for your provider:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-2.png)
 
-<% if (account.userName) { %>
-3. Download the metadata from <a href="https://${account.namespace}/samlp/metadata/${account.clientId}">https://${account.namespace}/samlp/metadata/${account.clientId}</a> and upload it here.
-<% } else { %>
-3. Download the metadata from `https://${account.namespace}/samlp/metadata/${account.clientId}` and upload it here.
-<% } %>
+3. Download your metadata document from <% if (account.userName) { %><https://${account.namespace}/samlp/metadata/${account.clientId}><% } else { %>`https://${account.namespace}/samlp/metadata/${account.clientId}`<% } %>.
+
+  Click **Choose File** and browse to the metadata document you just downloaded. Click **Next Step**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-3.png)
 
-4. The SAML Provider was created. You can close the modal.
+4. Verify your provider information and click **Create**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-4.png)
 
-## Creating Roles
+## Create a Role
 
-Now, you have to create a role that will have one or more policies associated. You can create as many roles as you want.
+Now create a role with one or more associated policies. (You can create as many roles as required.)
 
-1. Go to **Roles**.
+1. On the AWS IAM Console, click on **Roles** in the left menu and then click **Create New Role**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-5.png)
 
-2. Click on **Create New Role** and give it an arbitrary name.
+2. Provide a name for your new role and click **Next Step**:
 
   ![](/media/articles/aws-api-setup/aws-api-setup-6.png)
 
-3. Select or enter the following values.
+3. Select **Role for Identity Provider Access**. Then select **Grant API access to SAML providers** and click **Next Step**:
```

Comment:
I previously went through all of the updates, did something change? It
looks correct to me but want to be sure if there is something specific that
I look at it.

On Tue, Nov 17, 2015 at 9:40 AM, Nathan Totten <notifications@github.com>
wrote:

> In articles/aws-api-setup.md
> <https://github.com/auth0/docs/pull/562#discussion_r45066530>:
>
> >
> >    ![](/media/articles/aws-api-setup/aws-api-setup-6.png)
> >
> > -3. Select or enter the following values.
> > +3. Select **Role for Identity Provider Access**. Then select **Grant API access to SAML providers** and click **Next Step**:
>
> @chriskeyser <https://github.com/chriskeyser> can you verify this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/auth0/docs/pull/562/files#r45066530>.
>